app/Http/Controllers/Licenses/LicensesController.php

```diff
@@ -266,4 +266,10 @@ public function getClone($licenseId = null)
         ->with('item', $license)
         ->with('maintained_list', $maintained_list);
     }
+    public function getLicenseList(){
```

Comment:
What's the advantage of using `DB::table('licenses')->get()` and this method versus using something like:

`$licenses = License::get();`